StorageUtility/Linux/StorageUtility.cpp

```diff
@@ -210,4 +211,11 @@ void CloseDeviceHandle(DeviceHandle& Handle)
     }
 }
 
+eErrorCode GetPhysicalDiskNumber(DeviceHandle Handle, U32& PhysicalDiskNumber)
+{
+    //! TODO Use Device path on Linux system instead PhysicalDiskNumber
+
+    return(eErrorCode::None);
```

Comment:
If change return value to NoSupport then we can get drives beacause check condition of DriveEnumerateAta/Scsi is not None